README.md

```diff
@@ -171,6 +171,74 @@ this is loaded as it starts.
 If you want to run tests without running the `zipgateway` binary provided by Silicon Labs you
 can configure `run_zipgateway_bin` to be `false`:
 
+
+```elixir
+config :grizzly,
+  run_zipgateway_bin: false
+```
+
+#### Configuring zipgateway
+
+
+The `zipgateway` binary is passed a configuration file named `zipgateway.cfg`. This has configuration
+parameters around networking and setting device specific information.
+
+Supported configuration fields are:
+
+* `:unsolicited_destination_ip6` -  IPv6 string address defaults to `"fd00:aaaa::2` and should work for
```

Comment:
A lot of these fields seem like they should be supplied some other way or not allowed to be configured. For example, the IPv6 addresses are hardcoded like here:

https://github.com/smartrent/grizzly/blob/47f376203e7770807088f4b928873e2aeba5fb08/lib/grizzly.ex#L70

and the DTLS shared key is:

https://github.com/smartrent/grizzly/blob/49ad7da79c13fd0f3d3a5f200147d3b403f0d235/lib/grizzly/client/dtls.ex#L66

Reply:
Another idea would be to keep the options, but to raise exceptions if the user tries to set them. Then when you could tackle how to make these settings consistent with the currently hardcoded locations elsewhere at a later time.

Reply:
Yeah, I did not think about that. I think limiting options is a good idea. However, I would like to keep the tunscript path an option as with new tunscript that works with 7.11 in smartrent hub hasn't been friendly to the 2.81 silicon labs zipgateway and having the path will allow the old tunscript to be used.